cypress/support/commands.js

```diff
@@ -263,6 +238,56 @@ Cypress.Commands.add(
   },
 );
 
+Cypress.Commands.add('deleteRsvp', (eventId, userId) => {
+  const kickMutation = {
+    operationName: 'deleteRsvp',
+    variables: {
+      eventId,
+      userId,
+    },
+    query: `mutation deleteRsvp($eventId: Int!, $userId: Int!) {
+      deleteRsvp(eventId: $eventId, userId: $userId)
+    }`,
+  };
+  return cy.authedRequest({
+    method: 'POST',
+    url: 'http://localhost:5000/graphql',
+    body: kickMutation,
+  });
+});
+
+Cypress.Commands.add('authedRequest', (options) => {
+  return cy.request({
+    ...options,
+    headers: {
+      Authorization: `Bearer ${window.localStorage.getItem('token')}`,
+    },
+  });
+});
```

Comment:
This seems to be duplicate of lines further in the file: https://github.com/freeCodeCamp/chapter/blob/6618f6d10eb0f27ce6933e60a668b3b6b9b63ae9/cypress/support/commands.js#L259-L266

Reply:
That's what I get for trusting git rebase. Thanks!